pacgraph-tk

```diff
@@ -8,9 +8,21 @@ from multiprocessing import Process
 import sys, threading, queue, time
 
 if not dev:
-    import imp
-    imp.load_source('pacgraph', '/usr/bin/pacgraph')
-import pacgraph
```

Comment:
If `dev = True` is set to enable developement mode, `import pacgraph` from local folder should work. This MR moves all the importing of pacgraph into the  `if not dev:` branch. Which renders the development mode broken (with no import).

Instead of hard coding the dev mode, we could first try to `import pacgraph` locally and catch the `ImportError` and only then attempt to do the import magic.